python/tk_nuke_writenode/handler.py

```diff
@@ -1346,6 +1346,7 @@ def __update_render_path(self, node, force_reset=False, is_proxy=False):
                             just update the normal render path.
         :returns:           The updated render path
         """
+        self._app.logger.info("__update_render_path %r reset: %r" % (node, force_reset))
```

Comment:
Forget to remove this after debugging?

Reply:
ah yeah! I thought I'd removed all that in source tree, thanks!